### PR TITLE
Added ES6 default parameters.

### PR DESCRIPTION
**Pull request by olegon**
I'm training ES6 and Github workflow. :}

- Added _default parameters_ in __optCurrency function__;
- Added _.map()_ in __formatRates function__.

### CODE REVIEW

File: src/options/optCurrency.js [hunk 1]
```diff
@@ -42,27 +42,23 @@ function getCountryIcon(countryInitial) {
 }
 
 function formatRates(rates) {
-  const ratesArray = [];
-  Object.keys(rates).forEach((rate) => {
-    ratesArray.push(`${getCountryIcon(rate)}  ${rate} ${chalk.green(rates[rate])}\n-------------\n`);
-  });
-  return ratesArray.join('');
+  return Object.keys(rates)
+          .map(rate => `${getCountryIcon(rate)}  ${rate} ${chalk.green(rates[rate])}\n-------------\n`)
```
Comment (jmpas) [inline src/options/optCurrency.js]: I'm using two space indentation for method chaining.
[example here](https://github.com/gabrielgodoy/getme/blob/master/src/options/optWeather.js#L65).

But it's up to you guys, @gabrielgodoy :)

Comment (olegon) [reply src/options/optCurrency.js]: Hm, ok. I'll fix it later. Thank you! :} I wanna add some new features too.

**Comment by gabrielgodoy** [comment]
Ok, cool, seems like a good refactor to me.

I will put the 'work in progress' label for now. And when you put the indentation to two spaces for method chain, I remove the WIP label and merge.

I will try to add this rule on eslint for two spaces in chain methods.

**Comment by olegon** [comment]
@gabrielgodoy Added two spaces identation on method chain.

**Pull request merged**